src/quo/components/counter/collectible_counter/style.cljs

```diff
@@ -0,0 +1,72 @@
+(ns quo.components.counter.collectible-counter.style
+  (:require
+    [quo.foundations.colors :as colors]))
+
+(defn- get-background-color
+  [{:keys [status theme]}]
+  (case status
```

Comment:
there are a few misues of the resolve-color method. Mostly the value you should be passing there is a keyword, i.e a custom-color, a network color or a socials color, it can also be :danger or :success. however for these variable colors (e.g colors/white-opa-70) they don't need to be resolved as we already have the correct value.

This block should be:
```
:default (colors/theme-colors  colors/white-opa-70 colors/neutral-95-opa-70 theme)
    :error   (colors/resolve-color :danger theme 10)
    (colors/theme-colors  colors/white-opa-70 colors/neutral-95-opa-70 theme)

```
resolve colour will also handle the theming, so for example the use case of :danger we don't need to wrap it in theme-colors method as it is already done 👍